internal/services/netapp/netapp_volume_resource.go

```diff
@@ -221,6 +221,12 @@ func resourceNetAppVolume() *pluginsdk.Resource {
 				},
 			},
 
+			"snapshot_directory_visible": {
```

Comment:
I think this would be better as
```suggestion
			"snapshot_directory_is_visible": {
```

Reply:
This property was already there for at least two months, it just shifted due to addition of new properties. Not sure we should change this at this point and introduce a breaking change so soon for this.